### PR TITLE
feat: add generators from react-cmf

### PR DESCRIPTION
**Pull request by jmfrancois**
you will have two generators and I will not have to create a new repository here :)

yo talend:react-cmf
yo talend:react-component

both are really great and they are tested ;)

### CODE REVIEW

File: generators/react-cmf/templates/editorconfig [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+[*]
+indent_style = space
+indent_size = 2
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+[*.md]
+trim_trailing_whitespace = false
```
Comment (talend-glorieux) [inline generators/react-cmf/templates/editorconfig]: It might be a good thing to use the same config files for all the sub generators.

**Comment by jmfrancois** [comment]
Ok I'm stuck with tests on travis. they fails on travis only. Have to find why

**Comment by talend-glorieux** [comment]
@jmfrancois Do you want me to take a look at it?

**Comment by talend-glorieux** [comment]
Green tests :smile:

**Pull request merged**